packages/ubuntu_desktop_installer/lib/pages/allocate_disk_space_page.dart

```diff
@@ -126,8 +126,12 @@ class _AllocateDiskSpacePageState extends State<AllocateDiskSpacePage> {
                     final storageResponse =
                         await client.setGuidedStorage(choice);
 
-                    Navigator.pushNamed(context, Routes.writeChangesToDisk,
-                        arguments: storageResponse.config);
+                    await client.confirm('/dev/tty1');
+
+                    // Navigator.pushNamed(context, Routes.writeChangesToDisk,
+                    //     arguments: storageResponse.config);
+                    Navigator.of(context)
+                        .pushNamed(Routes.installationComplete);
```

Comment:
That's not right. The new page should go at the end of the current workflow, which is after the "Choose your look" page.